processor/queuedprocessor/queued_processor.go

```diff
@@ -149,6 +152,7 @@ func (sp *queuedSpanProcessor) Shutdown() error {
 
 func (sp *queuedSpanProcessor) processItemFromQueue(item *queueItem) {
 	startTime := time.Now()
+	statsTags := processor.StatsTagsForBatch(sp.name, processor.ServiceNameForNode(item.td.Node), item.td.SourceFormat)
```

Comment:
do you really need this one here? I think you were going to move it but then ended up with two.

Reply:
whoops